src/ray/gcs/gcs_client/service_based_accessor.cc

```diff
@@ -24,7 +24,7 @@ ServiceBasedJobInfoAccessor::ServiceBasedJobInfoAccessor(
     : client_impl_(client_impl) {}
 
 Status ServiceBasedJobInfoAccessor::AsyncAdd(
-    const std::shared_ptr<JobTableData> &data_ptr, const StatusCallback &callback) {
+    const std::shared_ptr<rpc::JobTableData> &data_ptr, const StatusCallback &callback) {
```

Comment:
To minimize changes like this, we can add `using ray::rpc;` at the beginning of this file. (It should be OK because it's a .cc file.)

Reply:
updated.